css/css-shapes/spec-examples/shape-outside-018.html

```diff
@@ -38,12 +39,12 @@
     <script src="/resources/testharnessreport.js"></script>
     <script src="support/spec-example-utils.js"></script>
     <script>
-    function checkFloats() {
+    document.fonts.ready.then(()=> {
```

Comment:
This needs `setup` and `done` calls too, doesn't it?

Reply:
Oops, yes it does. It passing was luck since it only has 1 subtest.